portal.go

```diff
@@ -808,7 +817,13 @@ func (portal *Portal) HandleMatrixMessage(evt *event.Event) {
 	}
 	if err != nil {
 		portal.log.Errorln("Error sending to iMessage:", err)
-		portal.sendErrorMessage(evt, err, false)
+		unsupported := false
+		certain := false
+		if strings.HasPrefix(err.Error(), "size_limit_exceeded") {
```

Comment:
The proper way to do this would be to define

```go
var ErrSizeLimitExceeded = ipc.Error{Code: "size_limit_exceeded"}
```

somewhere (as a global variable) and then

```suggestion
		if errors.Is(err, ErrSizeLimitExceeded) {
```